enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/types/embed.ts

```diff
@@ -80,6 +80,7 @@ export interface QuestionEmbedOptions {
 
   // parameters
   initialSqlParameters?: SqlParameterValues;
+  hiddenParameters?: string[];
```

Comment:
We support it on type level, but the UI visibility control is hidden currently (for 57)